src/calculateRentalCost.js

```diff
@@ -29,6 +29,28 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const costDay = 40;
+  const weekDiscount = 50;
+  const threeDayDiscount = 20;
+  const week = 7;
+  const threeDay = 3;
+  let total = 0;
+
+  if (days >= week) {
+    total = costDay * days - weekDiscount;
```

Comment:
```suggestion
    return total - bigDiscount;
```
There is no need to reassign. Just return the full price